litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts

```diff
@@ -1,31 +1,16 @@
 export default [
```

Comment:
Please raise this changes in a different PR.

Reply:
It is required..